test/evergreen.yml

```diff
@@ -44,7 +44,38 @@ functions:
     command: shell.exec
     params:
       script: |
+        set -o errexit
+        set -o verbose
         git clone https://github.com/wiredtiger/mongo-tests
+  "fetch mongodb repo" :
+    command: shell.exec
+    params:
+      script: |
+        set -o errexit
+        set -o verbose
+        git clone https://github.com/mongodb/mongo
+  "merge mongodb wt" :
+    command: shell.exec
+    params:
+      script: |
+        set -o errexit
+        set -o verbose
+        cp -a wiredtiger mongo/src/third_party/
```

Comment:
I guess you tested this clean copy functioned well? The vendoring tools have a few files as exceptions when copying, maybe they are not necessary in this case.

Reply:
I did, I added it in the comment section of [WT-7343](https://jira.mongodb.org/browse/WT-7343).